### PR TITLE
Problem: Voting Task and Network Upgrade Issue

### LINKED ISSUE #295: Problem: Fix logic for MsgVote and Network Upgrade task
Being worked in #294

### PR DESCRIPTION
**Pull request by cdc-Hitesh**
Fixes #295 
Changes:
- Network Upgrade check changes
- MsgVote Check changes

### CODE REVIEW

**Review by foreseaz — APPROVED** [review]
LGTM

File: infrastructure/httpapi/handlers/crossfire.go [hunk 1]
```diff
@@ -136,6 +136,13 @@ func (handler *Crossfire) getValidatorStats(operatorAddress string, dbValidator
 			finalValidatorStats.CommitCountPhase2 = validatorStatRow.Value
 		case crossfire_constants.PHASE_1N2_COMMIT_PREFIX:
 			finalValidatorStats.CommitCountPhase1n2 = validatorStatRow.Value
+		case crossfire_constants.VOTED_PROPOSAL_ID:
+			{
+				finalValidatorStats.TaskVote = crossfire_constants.INCOMPLETED
```
Comment (calvinaco) [inline infrastructure/httpapi/handlers/crossfire.go:141]: If the participant does not vote, there will be no row inside validator stat right? So this switch case will never reached, and the TaskVote will not be `INCOMPLETED`?

Comment (calvinaco) [reply infrastructure/httpapi/handlers/crossfire.go:141]: @cdc-Hitesh

Comment (cdc-Hitesh) [reply infrastructure/httpapi/handlers/crossfire.go:141]: You are absolutely correct. Spot on. I am changing this.

Also @calvinaco  I happen to find one another interesting issue. Pushing the changes. Will update you

**Pull request closed without merging**